config/initializers/secure_headers.rb

```diff
@@ -32,11 +32,11 @@
   google_doubleclick = %w[*.doubleclick.net *.googleads.g.doubleclick.net *.ad.doubleclick.net *.fls.doubleclick.net stats.g.doubleclick.net]
   google_apis        = %w[*.googleapis.com https://fonts.googleapis.com]
 
-  facebook  = %w[*.facebook.com *.connect.facebook.net]
+  facebook  = %w[*.facebook.com *.facebook.net *.connect.facebook.net]
   govuk     = %w[*.gov.uk www.gov.uk]
   hotjar    = %w[*.hotjar.com vc.hotjar.io wss://*.hotjar.com]
   jquery    = %w[code.jquery.com]
-  pinterest = %w[*.pinterest.com ct.pinterest.com *.s.pinimg.com]
+  pinterest = %w[*.pinterest.com ct.pinterest.com *.pinimg.com *.s.pinimg.com]
```

Comment:
I would have expected `*.pinimg.com` to match `*.s.pinimg.com` - sometimes I think it takes a while for CSP changes to propagate as well 🤷‍♂️

Reply:
Yeah, I'd have thought so too. I compared it to the list before yesterday's changes and this is how it was before.

Reply:
Agree, will remove the `s` prefix one